Makefile

```diff
@@ -16,6 +16,7 @@
 # but csi-sanity has its own build rules and only mock-driver gets
 # published as a container image.
 CMDS=mock-driver
+export GOFLAGS=-mod=vendor
```

Comment:
Strictly speaking, we currently don't need to set `GOFLAGS=-mod=vendor` because when building in `GOPATH`, go 1.12 still defaults to non-module builds and then uses `vendor`. I checked with strace. But I agree that it makes sense to already add `GOFLAGS=-mod=vendor` because then we won't forget to add it when updating to 1.13 where modules are on by default.

How to do it is a bit trickier. The way it is done here overrides whatever `GOFLAGS` the developer may have in his environment. `make GOFLAGS=...` can be used to override this default.

I think I prefer to not set GOFLAGS for developers. Just set it for CI builds, by adding it to `release-tools/travis.yml` and `release-tools/prow.sh`.